packages/commercetools/theme/components/profile/PasswordResetForm.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+  <ValidationObserver v-slot="{ handleSubmit, reset }">
+    <form class="form" @submit.prevent="handleSubmit(submitForm(reset))">
+      <ValidationProvider rules="required" v-slot="{ errors }" class="form__element">
+        <SfInput
+          data-cy="my-profile-input_currentPassword"
+          v-model="form.currentPassword"
+          type="password"
+          name="currentPassword"
+          label="Current Password"
+          required
+          :valid="!errors[0]"
+          :errorMessage="errors[0]"
+        />
+      </ValidationProvider>
+      <div class="form__horizontal">
+        <ValidationProvider rules="required|password" v-slot="{ errors }" vid="password" class="form__element">
+          <SfInput
+            data-cy="my-profile-input_newPassword"
+            v-model="form.newPassword"
+            type="password"
+            name="newPassword"
+            label="New Password"
+            required
+            :valid="!errors[0]"
+            :errorMessage="errors[0]"
+          />
+        </ValidationProvider>
+        <ValidationProvider rules="required|confirmed:password" v-slot="{ errors }" class="form__element">
+          <SfInput
+            data-cy="my-profile-input_repeatPassword"
+            v-model="form.repeatPassword"
+            type="password"
+            name="repeatPassword"
+            label="Repeat Password"
+            required
+            :valid="!errors[0]"
+            :errorMessage="errors[0]"
+          />
+        </ValidationProvider>
+      </div>
+      <SfButton data-cy="my-profile-btn_update-password" class="form__button">Update password</SfButton>
+    </form>
+  </ValidationObserver>
+</template>
+
+<script>
+import { ref } from '@vue/composition-api';
+import { ValidationProvider, ValidationObserver } from 'vee-validate';
+import { SfInput, SfButton } from '@storefront-ui/vue';
+
+export default {
+  name: 'PasswordResetForm',
+
+  components: {
+    SfInput,
+    SfButton,
+    ValidationProvider,
+    ValidationObserver
+  },
+
+  setup(_, { emit }) {
+    const clearForm = () => ({
```

Comment:
**question** isn't this implying that we're clearing the `form` not returning its default values?